src/main/java/net/minecraftforge/client/extensions/common/IClientItemExtensions.java

```diff
@@ -57,6 +59,20 @@ default Font getFont(ItemStack stack, FontContext context)
         return null;
     }
 
+    /**
+      * This method returns an ArmPose that can be defined using the {@link net.minecraft.client.model.HumanoidModel.ArmPose#create(String, boolean, IForgeArmPose)} method.
+      * This allows for creating custom item use animations.
+      *
+      * @param entityLiving The entity holding the item
+      * @param hand         The hand the ArmPose will be applied to
+      * @param itemStack    The stack being held
+      * @return A custom ArmPose that can be used to define movement of the arm
+      */
+            @Nullable
```

Comment:
Formatting: extra whitespace.
```suggestion
    @Nullable
```